env/env.cc

```diff
@@ -24,6 +24,8 @@ Env::~Env() {
 
 std::string Env::PriorityToString(Env::Priority priority) {
   switch (priority) {
+    case Env::Priority::GC:
```

Comment:
I think using the existing priorities is enough, don't need to add a special one.